content/docs/reference-react-dom-server.md

```diff
@@ -15,31 +15,31 @@ import ReactDOMServer from 'react-dom/server';
 var ReactDOMServer = require('react-dom/server');
 ```
 
-## Overview
+## Resumen
 
-The following methods can be used in both the server and browser environments:
+Los siguientes métodos se pueden utilizar tanto en el servidor como en el entorno del navegador:
 
 - [`renderToString()`](#rendertostring)
 - [`renderToStaticMarkup()`](#rendertostaticmarkup)
 
-These additional methods depend on a package (`stream`) that is **only available on the server**, and won't work in the browser.
+Estos métodos adicionales dependen de un paquete (`stream`) que **solo estan disponibles en el servidor**, y no funcionara en el navegador.
 
 - [`renderToNodeStream()`](#rendertonodestream)
 - [`renderToStaticNodeStream()`](#rendertostaticnodestream)
 
 * * *
 
-## Reference
+## Referencia
 
 ### `renderToString()`
 
 ```javascript
 ReactDOMServer.renderToString(element)
 ```
 
-Render a React element to its initial HTML. React will return an HTML string. You can use this method to generate HTML on the server and send the markup down on the initial request for faster page loads and to allow search engines to crawl your pages for SEO purposes.
+Renderiza un elemento React a su HTML inicial. React devolverá HTML en una cadena de texto. Puede usar este método para generar HTML en el servidor y enviar el marcado en la solicitud inicial para que las páginas se carguen más rápido y para permitir que los motores de búsqueda rastreen sus páginas con fines de SEO.
```

Comment:
Should be `puedes`. I'd put `para que las páginas se carguen más rápido y permitir que los motores de búsqueda...` because the second `para` refers to the pages too.